scripts/azureml-assets/azureml/assets/__init__.py

```diff
@@ -7,12 +7,12 @@
     Config,
     DEFAULT_ASSET_FILENAME,
     ModelConfig,
-    ModelDownloadType,

```

Comment:
Why are we deleting this? I remember @haritha-bendapudi facing an issue in the pipeline and having to send out another PR to include the imports. 

@haritha-bendapudi Could you please comment?

Reply:
Not in this pipeline @skasturi . In the ADO pipeline, they replace this __init__.py with another test __init__.py. Anything that is removed from __init__.py in this file has to be deleted/added in https://msdata.visualstudio.com/Vienna/_git/azureml-asset?path=/scripts/replace/__init__.py&version=GBmain&_a=contents